widgets/ChooseWorkerAccountButton.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace humhub\modules\tasks\widgets;
+
+
+use humhub\components\Widget;
+use humhub\modules\tasks\models\Task;
+use humhub\modules\xcoin\models\Account;
+use humhub\widgets\Button;
+
+class ChooseWorkerAccountButton extends Widget
+{
+    /**
+     * @var Task
+     */
+    public $task;
+
+    /**
+     * @inheritdoc
+     */
+    public function run()
+    {
+        $workerAccounts = Account::findAll([
+            'user_id' => $this->task->taskAssignedUsers[0],
```

Comment:
(:+1: ) I was wondering too about this issue, but if we change the entire module to support one assigned user and one responsible user for a specific task , it would took a while to be achieved regarding to time pressure and may have a real impact on functionalities, and also as i have understood from karim for the moment we'll stick with one assigned and one responsible and in the future we'll come back to multiple users, so that i've hardcoded some bunch of code but in other methods that i introduced i've take into consideration that we'll have multiple user in other version.

Reply:
ok. 
such things just tend to lead to problems and confusion in the future - and also know that @KarimChabrak is not aware of such details... you and we need to be aware of such details to make sure we can build working software.

Reply:
but I see your point and thoughts! so: great, good job! 
let's see how it goes and will be used.